packages/fxa-settings/src/lib/cache.ts

```diff
@@ -10,6 +11,7 @@ export interface OldSettingsData {
   sessionToken: hexstring;
   alertText?: string;
   displayName?: string;
+  email?: string;
```

Comment:
i know it's there but we don't use it. this was meant to be the minimal used subset